doc/manuals/user/update_operators.md

```diff
@@ -238,12 +238,114 @@ For instance, if the preexisting value of attribute A in entity E is `[1, 2, 3]`
 POST /v2/entities/E/attrs/A
 {
   "value": { "$pullAll": [2, 3] },
-  "type": "Number"
+  "type": "Array"
 }
 ```
 
 would change the value of attribute A to `[1]`.
 
+### `$set`
+
+To be used with attributes which value is an object to add/update a sub-key in the
+object without modifying any other sub-keys.
+
+For instance, if the preexisting value of attribute A in entity E is `{"X": 1, "Y": 2}` the
+following request:
+
+```
+POST /v2/entities/E/attrs/A
+{
+  "value": { "$set": {"Y": 20, "Z": 30} },
+  "type": "Object"
+}
+```
+
+would change the value of attribute A to `{"X": 1, "Y": 20, "Z": 30}`.
+
+For consistence, `$set` can be used with values that are not an object, such as:
+
+```
+POST /v2/entities/E/attrs/A
+{
+  "value": { "$set": "foo" },
+  "type": "Object"
+}
+```
+
+which has the same effect than a regular update, i.e.:
+
+```
+POST /v2/entities/E/attrs/A
+{
+  "value": "foo",
+  "type": "Object"
+}
+```
+
+We don't recommend this usage, as the regular update is simpler.
+
+### `$unset`
+
+To be used with attributes which value is an object to remove a sub-key from the
+object without modifying any other sub-keys.
+
+For instance, if the preexisting value of attribute A in entity E is `{"X": 1, "Y": 2}` the
+following request:
+
+```
+POST /v2/entities/E/attrs/A
+{
+  "value": { "$unset": {"X": 1} },
+  "type": "Object"
+}
+```
+
+would change the value of attribute A to `{"Y": 20}`.
+
+The actual value of the sub-key used with `$unset` is not relevant. A value of 1 is recommented
+for simplicity but the following request would also work and would be equivalent to the one above:
+
```

Comment:
Reading this, I have a doubt, what if executes the following request?

```
 POST /v2/entities/E/attrs/A
 {
   "value": { "$unset": {"X": 10} },
   "type": "Object"
 }
```

Reply:
It is also the same as above. The case is covered by the *the actual value of the sub-key used with `$unset` is not relevant* included above.

Reply:
NTC